usbreset.8.in

```diff
@@ -0,0 +1,39 @@
+.\"SPDX-License-Identifier: GPL-2.0
+.\"Copyright (c) 2019 Fabian Baumanis <fabian.baumanis@suse.com>
+.TH usbreset 8 "23 April 2019" "usbutils-VERSION" "Linux USB Utilities"
+.IX usbreset
+.SH NAME
+usbreset \- reset USB devices
+.SH SYNOPSIS
+.B usbreset
+[
+.I options
+]
+.SH DESCRIPTION
+.B usbreset
+is a utility for resetting USB devices via the USBDEVFS_RESET ioctl.
+
```

Comment:
Please add the huge problem that most USB hubs do not work with this program and you are on your own if it does, this is merely a sample test program for some people who happen to be lucky to be able to use.

I really didn't want to add this utility in the first place, given that it does not work well because it can not work well due to the USB spec not requiring this functionality.